projects/mydatepicker-ngcc/e2e/protractor.conf.js

```diff
@@ -13,10 +13,7 @@ exports.config = {
     './src/**/*.e2e-spec.ts'
   ],
   capabilities: {
-    browserName: 'chrome',
-    chromeOptions: {
-      args: [ "--headless", "--disable-gpu"]
-    }
+    'browserName': 'chrome'
```

Comment:
Is this intended?

Reply:
These are from #385 (only the last commit is relevant for this PR).